cram/cram_io.c

```diff
@@ -1893,15 +1893,29 @@ static int cram_populate_ref(cram_fd *fd, int id, ref_entry *r) {
     char *ref_path = getenv("REF_PATH");
     SAM_hdr_type *ty;
     SAM_hdr_tag *tag;
-    char path[PATH_MAX], path_tmp[PATH_MAX];
+    char path[PATH_MAX], path_tmp[PATH_MAX], cache[PATH_MAX];
     char *local_cache = getenv("REF_CACHE");
     mFILE *mf;
 
     if (fd->verbose)
 	fprintf(stderr, "cram_populate_ref on fd %p, id %d\n", fd, id);
 
-    if (!ref_path || *ref_path == 0)
+    if (!ref_path || *ref_path == 0) {
```

Comment:
Can I suggest  ref_path[0] == '\0' rather than *ref_path == '0' I'm pretty sure that's checking if the first char is a 0 and this would make this clearer at a glance .